src/Components/Web.JS/package.json

```diff
@@ -13,7 +13,7 @@
   "devDependencies": {
     "@aspnet/signalr": "link:../../SignalR/clients/ts/signalr",
     "@aspnet/signalr-protocol-msgpack": "link:../../SignalR/clients/ts/signalr-protocol-msgpack",
-    "@dotnet/jsinterop": "3.0.0-preview6.19280.1",
+    "@dotnet/jsinterop": "https://dotnet.myget.org/F/aspnetcore-dev/npm/@dotnet/jsinterop/-/@dotnet/jsinterop-3.0.0-preview8.19367.2.tgz",
```

Comment:
Do we really want the URL here? That is, could we use `3.0.0-preview8.19367.2`?

Reply:
I don't know how the build infrastructure flows node package dependencies between Extensions and ASP.NET Core. I tried several things and this is the only one that worked for me.

I trust you and John as the build experts to suggest the appropriate way of doing this, as I'm not familiar with this part of the build.

As far as I know this is the only thing that works, so if you want to update it to the "proper" way, I'll be ok with the change, but listing the version alone simply doesn't work.

Reply:
:ok: it's probably related to where the specific version is available. Might be able to go back to a simple version after we push the preview 8 publicly.